qiskit_sphinx_theme/directives.py

```diff
@@ -38,7 +38,7 @@ class CardItemDirective(Directive):
         "tags": directives.unchanged,
     }
 
-    def run(self):
+    def run(self) -> list:
```

Comment:
These should go back to `list[T]`, where T is the type that you had before. The Paragraph node thing.